server/intelligence-service/README.md

```diff
@@ -32,6 +34,7 @@ After running the application, you can access the FastAPI API documentation at `
 ## Project Structure
```

Comment:
I don't think that you need to keep this up to date with each file we are having, we could remove it for now. Unless you think it makes sense. You are anyways saying in the next sections that the tests are in located in `tests/`.